claimsponsor/claimsponsor.go

```diff
@@ -135,13 +137,32 @@ func (c *ClaimSponsor) claim(ctx context.Context) error {
 				time.Sleep(c.waitOnEmptyQueue)
 				return nil
 			}
-			return fmt.Errorf("error calling getClaim with globalIndex %s: %w", claim.GlobalIndex.String(), err)
+			return fmt.Errorf("getFirstPendingClaim failed: %w", err)
 		}
-		txID, err := c.sender.sendClaim(ctx, claim)
+
+		claim.TxID, err = c.sender.sendClaim(ctx, claim)
 		if err != nil {
+			// AlreadyClaimed(): execution reverted (0x646cf558)
+			if strings.Contains(err.Error(), "execution reverted (0x646cf558)") {
```

Comment:
can we make it const?

Reply:
IMO, it would be better if we have computed the Keccak256 hash of the `AlreadyClaimed()` error and take the first 4 bytes and capture it in a variable, which then can be used in string comparison, wdyt?

```go
var alreadyClaimedRevertCode = crypto.Keccak256("AlreadyClaimed()")[:4]
```